manifests/apache.pp

```diff
@@ -1,101 +1,46 @@
 # Setups Apache for Katello development
 class katello_devel::apache {
 
-  include apache
+  include foreman::params
+
+  class { 'foreman::config::apache':
+    passenger               => false,
+    ssl                     => true,
+    ssl_cert                => $certs::apache::apache_cert,
+    ssl_key                 => $certs::apache::apache_key,
+    ssl_ca                  => $certs::apache::ca_cert,
+    ssl_chain               => $certs::apache::ca_cert,
+		app_root                => '/home/vagrant/foreman',
+		passenger_ruby          => $foreman::params::passenger_ruby,
+		priority                => $foreman::params::vhost_priority,
+		servername              => $foreman::params::servername,
+		serveraliases           => $foreman::params::serveraliases,
+		server_port             => $foreman::params::server_port,
+		server_ssl_port         => $foreman::params::server_ssl_port,
+		ssl_certs_dir           => '',
+		ssl_crl                 => '',
+		user                    => $foreman::params::user,
+		passenger_prestart      => false,
+		passenger_min_instances => 0,
+		passenger_start_timeout => 0,
+		foreman_url             => $foreman::params::foreman_url,
+		ipa_authentication      => false,
+    proxy_backend           => "http://0.0.0.0:${katello_devel::rails_port}/",
```

Comment:
```suggestion
    proxy_backend           => "http://localhost:${katello_devel::rails_port}/",
```

Reply:
Developers still access the dev server directly, which means it does not run on localhost.

Reply:
I hope they let it listen on `*` or `0.0.0.0` which means it still listens on localhost. I also wonder why they need to access the dev server directly if Apache is in front of it.